cfme/tests/cli/test_appliance_console_db_restore.py

```diff
@@ -72,19 +80,7 @@ def get_replicated_appliances_with_providers(temp_appliances_unconfig_funcscope_
     prior to running tests.
 
     """
-    appl1, appl2 = temp_appliances_unconfig_funcscope_rhevm
-    # configure appliances
-    appl1.configure(region=0)
-    appl1.wait_for_web_ui()
-    appl2.configure(region=99)
-    appl2.wait_for_web_ui()
-    # configure replication between appliances
-    appl1.set_pglogical_replication(replication_type=':remote')
-    appl2.set_pglogical_replication(replication_type=':global')
-    appl2.add_pglogical_replication_subscription(appl1.hostname)
-    # Add infra/cloud providers and create db backups
-    provider_app_crud(VMwareProvider, appl1).setup()
-    provider_app_crud(EC2Provider, appl1).setup()
+    appl1, appl2 = replicated_appliances_with_providers(temp_appliances_unconfig_funcscope_rhevm)
```

Comment:
This method is a fixture, and cannot be called directly (nor should it be imported above)